src/components/executive-education-2u/components/EnrollmentCompletedSummaryCard.jsx

```diff
@@ -22,13 +27,14 @@ const EnrollmentCompletedSummaryCard = () => {
         </Col>
         <Col xs={12} md={9}>
           <Card.Section>
-
             <div className="h3 mb-4">What happens next?</div>
             <div className="mb-3.5">
               <div className="mb-1.5 text-black-color">Notified by email</div>
               <div className="small mb-2 text-gray-500">
                 GetSmarter will email you when your course starts. Alternatively, you can visit your
-                GetSmarter learner dashboard for course status updates.
+                <Hyperlink destination={`${config.GETSMARTER_LEARNER_DASHBOARD_URL}?org_id=${orgId}`}>
+                  GetSmarter learner dashboard
+                </Hyperlink> for course status updates.
```

Comment:
nit/suggestion: In the event `enterpriseConfig.orgId` doesn't exist, should we avoid passing the `org_id` query parameter? Otherwise, I believe we'd pass `?org_id=undefined`.